### ISSUE TITLE
histogram(0) causes `DivideError: integer division error` 

### ISSUE THREAD

**Issue by goretkin** [JuliaPlots/Plots.jl#931]
`histogram(0)` raises `DivideError: integer division error` 

(I entered this unintentionally, but I suppose there could be a clearer, earlier error message)
Julia Version 0.5.2, Plots : 0.11.3

**Comment by daschw** [comment]
I cannot reproduce this. Which backend are you using?

**Comment by goretkin** [comment]
Hmm, perhaps I was mistaken. Can you try `histogram(100)`? That triggers it for me.

**Comment by daschw** [comment]
Yes, I can reproduce the error with `histogram(100)`
However, what would you expect from `histogram(100)`? If you look at the [Basics](https://juliaplots.github.io/basics/) section in the docs, you can see that `plot(n)` initializes a plot with `n` empty series for Integers `n`. Do you wish to initialize a plot with 100 empy histogram series? Usually input data is passed in the form of arrays.

**Comment by mkborregaard** [comment]
Can you think of any use case for this?

**Comment by goretkin** [comment]
I mentioned in my original message that perhaps there could be a clearer, more immediate error (as opposed to an error originating several calls deep).

But, no, I don't propose any interpretation for `histogram(100)`, and don't think it is a good idea to give it one (though I suppose you can make it consistent with the behavior that @daschw describes for `plot`)

**Comment by mkborregaard** [comment]
Yes it seems to be something in the histogram calculations that throws this error (`plot(100, st = :histogram)` throws it too). I'd say in general that error messages in Plots are ripe for improval! I've edited the title a little.
